src/unread/UnreadCards.js

```diff
@@ -52,7 +52,12 @@ class UnreadCards extends PureComponent<Props> {
         key: 'private',
         data: [{ conversations, ...restProps }],
       },
-      ...unreadStreamsAndTopics,
+      ...unreadStreamsAndTopics.filter(item => item.isPinned),
+      {
+        key: 'separator',
```

Comment:
You don't need to add a dummy section. There are several ways to rework that.
One is to create two sections, one is 'Pinned' the other is 'Rest'.
Also, we should consider showing the header titles.

Reply:
Then, we will loss the floating heading of `StreamItem`, if we go with `pinned` and `rest` as sections. Currently each stream unread are treated as different section, and one section for all PM unread.

So if we go with
```
SectionSeparatorComponent={SectionSeparatorBetween}
```
then separator will be shown after each stream.

 Or are you suggesting to have nested `SectionList`?.